openslides/core/urls.py

```diff
@@ -7,10 +7,15 @@
     url(r'^core/url_patterns/$',
         views.UrlPatternsView.as_view(),
         name='core_url_patterns'),
+
     url(r'^core/version/$',
         views.VersionView.as_view(),
         name='core_version'),
 
+    url(r'^angular_js/(?P<app>site|projector)/$',
```

Comment:
The termin `app` may cause missunderstandings (Django app or AngularJS app) so I propose to refer to `OpenSlidesApp` (which is `openslides_app` in lowercase writing:
```
url(r'^core/js_files/(?P<openslides_app>site|projector)/$'
```